test/export/test_export.py

```diff
@@ -1267,6 +1267,27 @@ def false_fn(x):
         )
         torch.export.export(M(), args)
 
+    def test_numpy(self):
+        class Foo(torch.nn.Module):
+            def forward(self, x):
+                a = x.numpy()
+                return x + x.numpy().sum()
+
```

Comment:
What's the strategy for capturing .numpy()? What does the graph look like?

Reply:
I will update the test with the graph. BUt main strategy is just porting: https://github.com/pytorch/pytorch/blob/main/torch/_dynamo/variables/tensor.py#L822